redwoodjs/redwood
# Renames GraphQL Healthcheck Path

The current GraphQL Healthcheck path referred to ApolloServer.

This changes the path to

`const HEALTH_CHECK_PATH = '/health'`

To test:

* Startup api, GraphQL
* Visit `http://localhost:8911/graphql/health`

Should get a pass or a fail. Or, if you set a custom `onHealthCheck` the result of that.

- [x ] Needs docs. See: https://github.com/redwoodjs/redwoodjs.com/pull/818
